readthedocs/projects/models.py

```diff
@@ -697,29 +691,20 @@ def all_active_versions(self):
         """
         return self.versions.filter(active=True)
 
-    def supported_versions(self, flat=True):
+    def supported_versions(self):
         """
         Get the list of supported versions.
         Returns a list of version strings.
         """
         if not self.num_major or not self.num_minor or not self.num_point:
```

Comment:
Yes, that sounds like a good idea! Either only one message saying "there is a newer version available" or a distinct message for updates on the minor and one for updates on the major version.

So that if you are on the docs for 1.2 and there are the versions 1.3 and 2.0 available it would show **"there is a newer version for 1.x"** and a second message with **"There is a new major release 2.x maybe you want to check it out"**